hack/verify-gen-crds.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+./hack/gen-crds.sh
+
+if ! git diff --exit-code config/crds.yml >/dev/null; then
+  echo "CRD verification - failed! Generated CRDs are out of date. Please run './hack/gen-crds.sh' and 'git add' the generated file(s)."
+  exit 1
+fi
```

Comment:
i think this should happen only in ci with a failure. it's a little weird for the script to gen things, then fail, to expect you to git add, then you have to rerun it.

Reply:
Thanks, moved to test-gh workflow